gradle.properties

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-ideaEdition = IC
+ideaEdition = IU
```

Comment:
@patflynn Why not keep this? IU is our main use case anyway. I personally always manually change this to IU when running locally.